### ISSUE TITLE
Error in Widget

### ISSUE THREAD

**Issue by liucoj** [Vxider/NiuDataWidget#13]
HI, I followed your installation guide but it result in an error, any fix?

![IMG_20A8F215FA5C-1](https://user-images.githubusercontent.com/14237462/214600876-47f247a4-b3f9-48fd-aadf-160085e77108.jpeg)

**Comment by liucoj** [comment]
any help?

**Comment by ClayMoreBoy** [comment]
密码错误？

**Comment by liucoj** [comment]
> 密码错误？

no, password is correct, seems something missing during auth. Country code parameter? I'm trying to run your script from Europe

**Comment by Vxider** [comment]
You can try modifying the CountryCode in the script.
https://github.com/Vxider/NiuDataWidget/blob/3c1a9217058942fe0ad7b6293c624a9334656713/NiuDataWidget.js#L803

**Comment by liucoj** [comment]
After editing country_code it seems working but some other errors came out,
can you provide more detailed steps to setup and display widget in lockscreen?
thank you
![Screenshot 2023-04-19 (10 21 29)](https://user-images.githubusercontent.com/14237462/233016564-687f75bc-c96a-4902-9638-3602a14d0077.png)

**Comment by liucoj** [comment]
this error came out when using one of your shortcuts
![IMG_448FDA675476-1](https://user-images.githubusercontent.com/14237462/233057697-a93cb6b3-77ea-4d44-9d4d-51362a9859ef.jpeg)

**Comment by liucoj** [comment]
on line 821 is correct this api call? "https://app-api.niu.com/v5/cmd/creat"
